input/drivers/android_input.c

```diff
@@ -638,53 +645,77 @@ static int android_check_quick_tap(android_input_t *android)
 }
 
 static INLINE void android_mouse_calculate_deltas(android_input_t *android,
-      AInputEvent *event,size_t motion_ptr)
+      AInputEvent *event,size_t motion_ptr,int source)
 {
-   /* Adjust mouse speed based on ratio
```

Comment:
I removed the code here because it didn't seem to accurately set the mouse speed - it made it significantly slower compared to what you'd expect it, based on the Android system mouse speed settings: https://github.com/libretro/RetroArch/pull/16203#issuecomment-1933118104

Since the user can adjust their mouse speed in Android settings, and/or (sometimes) within core options, I don't think it's really necessary